docs/xxenablecriusupport.md

```diff
@@ -0,0 +1,61 @@
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]EnableCRIUSupport
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+ This option enables and disables the support for Checkpoint Restore in User Space (CRIU) in the VM.
+
+ :fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:**
+
+ - If the CRIU support is enabled, Java security managers are unsupported. Enabling a security manager with `-Djava.security.manager`, `System.setSecurityManager` or other means throws `java.lang.UnsupportedOperationException`.
+ - When CRIU support is enabled, CRIUSECProvider is the only security provider in the checkpoint phase until restore.
+ - `-Xgcpolicy:balanaced` and `-Xgcpolicy:metronome` are disabled when this option is enabled.
+ - JITServer is disabled in the checkpoint phase when this option is implicitly enabled on restore.
+
+
+## Syntax
+
+        -XX:[+|-]EnableCRIUSupport
+
+| Setting               | Effect  | Default                                                                            |
+|-----------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+EnableCRIUSupport` | Enable  |                                                                                 |
+| `-XX:-EnableCRIUSupport` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>  |
+
+
+## Explanation
+
+When the option is enabled, the VM starts with the support for checkpoint and restore. The VM starts the JIT, GC, Security, and other internal components that make checkpoint and restore feasible. The option enables the use of `org.eclipse.openj9.criu.CRIUSupport` APIs. With CRIU support, you can take a checkpoint of the VM and restore the checkpointed image. Because a checkpoint can be restored on a different system, when the `-XX:+EnableCRIUSupport` option is specified, the VM generates only those compiled codes that are portable (see [`-XX:[+|-]PortableSharedCache`](https://www.eclipse.org/openj9/docs/xxportablesharedcache/)) to facilitate restoring of the checkpoint image in as many environments as possible. VM does not load any code from the SCC that is nonportable pre-checkpoint.
+
+Since checkpoint and restore generally happens on different environments with different behaviours, OpenJ9 performs compensations for such differences to ensure successful restore. For example, the VM compensates for the downtime between checkpoint and restore for elasped time sensitive APIs such as `System.nanotime()` and `java.util.Timer`.
```

Comment:
See Word Usage for "since" and "perform"
"happens" -> "happen"
Use of "checkpoint" as verb. In the first instance, you probably mean "checkpoint and restore processes". In the second instance, you probably mean "checkpoint and restore points"
"behaviours" -> "behaviors"
"elasped" -> "elapsed"